src/Silex/Application.php

```diff
@@ -272,10 +271,10 @@ public function after($callback, $priority = 0)
     {
         $app = $this;
 
-        $this->on(KernelEvents::RESPONSE, function (FilterResponseEvent $event) use ($callback, $app) {
-            if (!$event->isMasterRequest()) {
-                return;
-            }
+        $this->on(KernelEvents::RESPONSE, function (ResponseEvent $event) use ($callback, $app) {
```

Comment:
https://github.com/symfony/http-kernel/blob/v4.4.50/Event/FilterResponseEvent.php
```
/**
 * @deprecated since Symfony 4.3, use ResponseEvent instead
 */
```